docs/.vitepress/strings/en.json

```diff
@@ -302,7 +302,15 @@
               }
             },
             "cache": {
-              "text": "Cache"
+              "text": "Cache",
+              "items": {
+                "xcode-cache": {
+                  "text": "Xcode cache"
+                },
+                "generated-project": {
+                  "text": "Generated project"
```

Comment:
Naming is hard... `Cache > Generated project` reads odd. Should we call this "Module cache" or something similar, and then inside include an admonition saying that this is only for generated projects?